yfinance/base.py

```diff
@@ -45,6 +45,9 @@
 from .const import _BASE_URL_, _ROOT_URL_
 
 
+_empty_series = pd.Series()
```

Comment:
Why?
- cost of `pd.Series()` is negligible
- `_empty_series` is more characters